lib/far_mar/market.rb

```diff
@@ -0,0 +1,38 @@
+class FarMar::Market < FarMar::Base
+  attr_accessor :id, :name, :address, :city, :county, :state, :zip
+  FILE = ("support/markets.csv")
+
+  def initialize(attrs)
+   @id= attrs[0].to_i
+   @name = attrs[1]
+   @address = attrs[2]
+   @city = attrs[3]
+   @county = attrs[4]
+   @state = attrs[5]
+   @zip = attrs[6].to_i
+  end
+
+#refactor all ones like this
+  def vendors
+    FarMar::Vendor.by_market(@id)
+    # all_vendors = vendors.group_by do |obj|
+    #   obj.market_id
+    # end
+    # all_vendors[@id]
+  end
+
+  def self.find_by_state_name(state_name)
+    state_name = state_name.to_s
```

Comment:
Why did you decide to do this? Was there cases where the state name wasn't already a string?